src/components/sections/employeesCompaniesHistory/content/index.js

```diff
@@ -0,0 +1,20 @@
+const content = [
+  { image: require(`./images/1.svg`) },
+  { image: require(`./images/2.svg`) },
+  { image: require(`./images/3.svg`) },
+  { image: require(`./images/4.svg`) },
+  { image: require(`./images/5.svg`) },
+  { image: require(`./images/6.svg`) },
+  { image: require(`./images/7.svg`) },
+  { image: require(`./images/8.svg`) },
+  { image: require(`./images/9.svg`) },
+  { image: require(`./images/10.svg`) },
+  { image: require(`./images/11.svg`) },
+  { image: require(`./images/12.svg`) },
+  { image: require(`./images/13.svg`) },
+  { image: require(`./images/14.svg`) },
+  { image: require(`./images/15.svg`) },
+  { image: require(`./images/16.svg`) }
```

Comment:
Please give meaningful names to the files.

Reply:
Done